modules/core/src/main/scala/org/aossie/agora/model/Candidate.scala

```diff
@@ -1,6 +1,6 @@
 package org.aossie.agora.model
 
-case class Candidate(
+class Candidate(
```

Comment:
Good we could leave this like this. However if for any reason we see that having this as a class isn't appropriate we gonna convert it to a trait.